ui/src/components/utils/DateUtils.js

```diff
@@ -36,6 +36,15 @@ class DateUtils {
         return this.getLocalDate(currDt, currentTimezone, 'UTC');
     }
 
+    getDateMinusSevenDays(currentDate, currentTimezone) {
```

Comment:
can you rename the function since this is actually returning a boolean

Reply:
done